canonical/ubuntu.com
# Context footer bem

## Done
- updated the contextual footer partial from download and made it generic for all sections and BEMed it (I think)
- removed all the section based contextual_footer partials
- created files for every contextual footer in /shared/contextual_footers
- created a new scss for for the contextual footers
- removed all the old scss from the various files
- added some footer info for tablet now that it is for sale
- updated the generic sections with new footer info (about, support, legal, tv)
- removed several old filed that were no longer in use and confusing
## QA
1. review the new _contextual_footer.html partial
2. review the new _contentual_footer.scss file
3. click around the site in S/M/L to see if the footer looks very close to the existing site
4. look at various pages and see if the content is the same between live and the partial
## Issue / Card
- NOTE: if this is approved, we should create cards to look at promoting to vanilla or uvt
